### ISSUE TITLE
Bug in trivial.c test?

### ISSUE THREAD

**Issue by dabrahams** [jeffreykegler/libmarpa#89]
`nulled_symbols` is written but never read, and then it is unconditionally reported that each nullable event has been nulled.

**Comment by dabrahams** [comment]
Also, I think the only way `spurious_nulled_events` can be zero as expected at the end of the test is if `highest_symbol_id <= 0`.  Is this test even run?  Maybe I'm missing something.

**Comment by jeffreykegler** [comment]
Definitely strange goings on here.  I will investigate.

**Comment by jeffreykegler** [comment]
@dabrahams Thanks for pointing out this particular dog's lunch.  The whole event testing code in trivial.c was problematic and is now rewritten.  I think this came about when I was using this code to debug a specific problem.  Once the problem was solved and all tests passed, I moved on, without doing the required desk-check.

Absent feedback to the contrary, I will close this in a couple of days.
